script.js

```diff
@@ -213,7 +262,7 @@ function openDropdown(dropdownElement) {
  *
  */
 function updateDropdownLabel() {
-  // code goes here
+  dropdownLabelElement.innerHTML = semester;
```

Comment:
nit - `semester` is in an outer scope. You can avoid future confusion by passing it directly to this function.